### PR TITLE
Update version: agalwood.Motrix version 1.5.15

### PR DESCRIPTION
**Pull request by CoolPlayLin**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115393&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/a/agalwood/Motrix/1.5.15/agalwood.Motrix.locale.en-US.yaml [hunk 1]
```diff
@@ -1,28 +1,60 @@
...
+  - fix: upnp client is destroyed #662
+  - fix: macOS full screen mode no traffic light #663
+  - fix: Windows auto launch preference config lose #662
+  - fix: keyboard shortcut mapping error for Linux and Windows versions
+  - fix: too fast to shut down the engine, reduce error logs
+  - fix: too fast to toggle tracker syncing spinner
+  - refactor: improve macOS fullscreen mode usability
+  - refactor: remove title bar buttons spacer
+  - chore: Optimize Russian translation #686
+  更新日志
```
Comment (Trenly) [inline manifests/a/agalwood/Motrix/1.5.15/agalwood.Motrix.locale.en-US.yaml:42]: These should be moved to a separate locale file

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
>  Manifest: The filename must match the PackageIdentifier and ManifestType properties from the manifest. actual 'agalwood.Motrix.locale.zh-CN.yaml', expected 'agalwood.Motrix.locale.en-US.yaml'

File: manifests/a/agalwood/Motrix/1.5.15/agalwood.Motrix.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Created with Komac v1.9.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: agalwood.Motrix
+PackageVersion: 1.5.15
+PackageLocale: en-US
```
Comment (stephengillie) [inline manifests/a/agalwood/Motrix/1.5.15/agalwood.Motrix.locale.zh-CN.yaml:6]:
```suggestion
PackageLocale: zh-CN
```

**Comment by stephengillie** [comment]
Hi @CoolPlayLin,

Apologies for making a change to your PR. I mistakenly believed I was working with vedantmgoyal2009's PR.

**Review by stephengillie — APPROVED** [review]

**Comment by CoolPlayLin** [comment]
@stephengillie It doesn't matter

**Pull request merged**